webapp/components/wpt-runs.html

```diff
@@ -218,58 +232,61 @@
       async ready() {
         super.ready();
         this.loadData();
+        this._createMethodObserver('testRunsLoaded(testRuns, testRuns.*)');
       }
 
       async loadData() {
-        return this.load(this.loadRuns().then(testRuns => {
-          let browsers = new Set();
-          // Group the runs by their revision/SHA
-          let testRunsBySha = testRuns.reduce((accum, results) => {
-            browsers.add(results.browser_name);
-            if (!accum[results.revision]) {
-              accum[results.revision] = {};
-            }
-            if (!accum[results.revision][results.browser_name]) {
-              accum[results.revision][results.browser_name] = [];
-            }
-            accum[results.revision][results.browser_name].push(results);
-            return accum;
-          }, {});
+        return this.load(this.loadRuns());
```

Comment:
I don't see much benefit keeping this method now. In fact, it's probably too much abstraction/indirection (especially too many `load*` functions around here). We can just replace all `this.loadData` with `this.load(this.loadRuns())`, which is more consistent with `this.load(this.loadMoreRuns())`.

Reply:
Done.